scripts/cibuild/ConfigureVstsBuild.ps1

```diff
@@ -72,7 +72,17 @@ Function Update-VsixVersion {
     Write-Host "Updated the VSIX version [$oldVersion] => [$($root.Metadata.Identity.Version)]"
 }
 
+Function Print-GitLog {
+    $mdFolder = Join-Path $env:SYSTEM_DEFAULTWORKINGDIRECTORY (Join-Path 'MicroBuild' 'Output')
+    New-Item -ItemType Directory -Force -Path $mdFolder | Out-Null
+    $mdFile = Join-Path $mdFolder 'GitLog.md'
+    & git log -1 --pretty=format:' Author: %an%n Commit: [%H](https://github.com/NuGet/NuGet.Client/commit/%H)%n Message: %s' 2>&1  | Set-Content $mdFile
+    Copy-Item -Path $mdFile -Destination "\\scratch2\scratch\ragrawal\gitlog.md" 
```

Comment:
Why do you need to copy to this file share?  I assume it was only necessary for debugging purposes.  If it's still needed, the path should be an environment variable.